### PR TITLE
Added changes for missing API Functionalities

### PR DESCRIPTION
**Pull request by Himan10**
1. Added new fields in the accounts and jobs models
2. Combined some fields for better representation

### CODE REVIEW

**Comment by Himan10** [comment]
Issue: #19
Fix: Adding more fields to the Jobs and User model (Point 2 of _Things to add in the code_)

Parth will update the seed files once we get the confirmation on the above added fields.

File: apps/accounts/serializers.py [hunk 1]
```diff
@@ -8,14 +8,23 @@
...
     class Meta:
         model = User
-        fields = ["email", "name", "password", "password2", "user_type"]
+        fields = [
+            "email",
+            "name",
+            "password",
+            "password2",
+            "user_type",
+            "age",
```
Comment (hims1911) [inline apps/accounts/serializers.py:25]: age and gender we dont have to store in auth table

File: apps/accounts/views.py [hunk 2]
```diff
@@ -74,33 +73,37 @@ def verify_and_get_payload(token):
...
-        otp,secret = OTP.generate_secret_with_otp()
+        otp, secret = OTP.generate_secret_with_otp()
         user.otp_secret = secret
         user.save()
 
     # Send Email
-    if purpose=="verify":
-        subject="Verify your account"
+    if purpose == "verify":
+        subject = "Verify your account"
```
Comment (hims1911) [inline apps/accounts/views.py:97]: can we create a constant.py, where we can store all the hardcoded or formatted strings we are using

File: apps/accounts/views.py [hunk 3]
```diff
@@ -116,19 +119,21 @@ def post(self, request, format=None):
...
         dummy_data = {
             "user_id": user.id,
-            "name" : user.name,
+            "name": user.name,
             "email": user.email,
             "user_type": user.user_type,
-            "about": None
+            "age": user.age,
+            "gender": user.gender,
+            "about": None,
```
Comment (hims1911) [inline apps/accounts/views.py:132]: if we dont pass about then it will pick none automatically

File: apps/accounts/views.py [hunk 4]
```diff
@@ -149,9 +154,13 @@ def post(self, request, format=None):
             payload = GenerateToken.verify_and_get_payload(dummy_token)
             # print(payload)
         except InvalidToken as e:
-            return Response({"errors": {"token":str(e)}}, status=status.HTTP_401_UNAUTHORIZED)
+            return Response(
+                {"errors": {"token": str(e)}}, status=status.HTTP_401_UNAUTHORIZED
```
Comment (hims1911) [inline apps/accounts/views.py:158]: create common response.py 
where you can return errors and proper response by passing msg variable.

File: apps/accounts/views.py [hunk 5]
```diff
@@ -179,13 +188,14 @@ def post(self, request, format=None):
...
-                    {"token": token, "msg": "Login Success",'verify':True}, status=status.HTTP_200_OK
+                    {"token": token, "msg": "Login Success", "verify": True},
+                    status=status.HTTP_200_OK,
                 )
             else:
                 token = OTP_DummyToken(user)
                 return Response(
-                    {'msg':'User not verified','token':token,'verify':False},
-                    status=status.HTTP_200_OK
+                    {"msg": "User not verified", "token": token, "verify": False},
```
Comment (hims1911) [inline apps/accounts/views.py:197]: keep msgs in constants.py

File: apps/jobs/constants/values.py [hunk 6]
```diff
@@ -1,11 +1,21 @@
...
 
+STATUS_CHOICES = (
+    ("under-reviewed", "Under-Reviewed"),
+    ("shortlisted", "Shortlisted"),
+    ("accepted", "Accepted"),
+    ("rejected", "Rejected"),
+    ("on-hold", "On-Hold"),
+)
+
+GENDER = (("male", "Male"), ("female", "Female"), ("other", "Other"))
```
Comment (hims1911) [inline apps/jobs/constants/values.py:12]: I dont think so you have to keep all the case sensitive values.

while validating it or consuming it you can convert according to your use case.

File: apps/jobs/constants/values.py [hunk 7]
```diff
@@ -1,11 +1,21 @@
-"""This file contains constant which are also used in other parts of the jobs API code, 
+"""This file contains constant which are also used in other parts of the jobs API code,
```
Comment (hims1911) [inline apps/jobs/constants/values.py:1]: file name should be constants.py
if config 
configs.py
if msg or response response.py

**Review by hims1911 — CHANGES_REQUESTED** [review]

File: apps/accounts/serializers.py (hunk 1 above)
Comment (Himan10) [reply apps/accounts/serializers.py:25]: okay, so I'll accept these fields in the request body, but will only save this in `tbl_user_profile`

**Review by hims1911 — APPROVED** [review]

**Pull request merged**